client/components/tile-grid/tile.jsx

```diff
@@ -37,7 +37,7 @@ export default class extends React.PureComponent {
 		const tileClassName = classNames( 'tile-grid__item', className );
 
 		return (
-			<Card className={ tileClassName } href={ href } onClick={ onClick }>
+			<Card className={ tileClassName } href={ href } onClick={ onClick } tabIndex="-1">
```

Comment:
Removing this from the tab index because the focus shows up on the call to action button as well. The double tab seemed redundant.